### ISSUE TITLE
Crossed user invitation duplicated

### ISSUE THREAD

**Issue by jpbottinelli** [quarentime/quarentime-back-end#36]
When user A invites user B and user B invites user A, after user A accepts invitation, the endpoint /Users/Contacts for user A, sees the user B duplicated. Probably because there's no user id.

```
{
    "result": [
        {
            "name": "test_quarentime_user_2_1586127722223",
            "phone_number": "+11586127722223",
            "user_id": "WG6PT6X57nfMHQ6vPskbv24uQhx1",
            "date_added": "2020-04-05T23:05:21.577213Z",
            "status": "healthy",
            "pending": false
        },
        {
            "name": "test_quarentime_user_2_1586127722223",
            "phone_number": "+11586127722223",
            "date_added": "2020-04-05T23:03:51.616016Z",
            "status": "healthy",
            "pending": true
        }
    ],
    "request_id": "af45a25a-c5de-45ee-aa1e-a4be46ec7155"
}
```

**Comment by edmarsj** [comment]
The error here happens because each invite from each different user is a different record on the database. We need to know what is the desired behavior in this case:

A) When user B invites user A, it should notify him that there's a pending request from that user first and let him accept it, or
B) When user B invites user A, it should add A automatically and accept the current request, or
C) When user A or B accepts the request, both users are added and the spare request is destroyed.

**Comment by ale90bsas** [comment]
@edmarsj we should review this with POs.

From my perspective, A and C sound better.

Best,

**Comment by jpbottinelli** [comment]
I just put something similar to A in #31. I'd go with that option. If we don't create duplicated invitations in the first place, we avoid this problem

**Comment by edmarsj** [comment]
@jpbottinelli, @mapplics  I'll close this issue and fix this on #31
